src/components/POS/Classic/SelectedItemRow.vue

```diff
@@ -321,6 +323,26 @@ export default defineComponent({
       profileRateSetting: null as boolean | null,
     };
   },
+  watch: {
+    'row.batch': {
+      async handler(newBatch) {
+        if (newBatch) {
+          await this.getAvailableQtyInBatch();
+        } else {
+          this.availableQtyInBatch = 0;
+        }
+      },
+      immediate: true,
+    },
+    batchAdded: {
+      handler(newValue) {
+        if (newValue) {
+          this.isExapanded = true;
+        }
+      },
+      immediate: true,
+    },
+  },
```

Comment:
```suggestion
  watch: {
    'row.batch': {
      async handler(newBatch) {
        if (newBatch) {
          this.availableQtyInBatch = await this.getAvailableQtyInBatch();
          this.isExapanded = true;
        }
      },
      immediate: true,
    },
  },
```

Compress the two watchers into one